src/WattleScript.Interpreter/Execution/VM/FunctionBuilder.cs

```diff
@@ -237,6 +237,11 @@ public int Emit_Literal(DynValue value)
 					return AppendInstruction(new Instruction(OpCode.PushString, StringArg(value.String)));
 				case DataType.Range:
 				{
+					if (value.Range.To > Instruction.NumVal2Min && value.Range.To < Instruction.NumVal2Max)
```

Comment:
`>=` , `<=` ?

Reply:
Fixed in 8b1b09eedad6317bdb85325807c3bf721dd271c6